package/gluon-web-logging/luasrc/lib/gluon/web/model/admin/logging.lua

```diff
@@ -0,0 +1,49 @@
+local uci = require('simple-uci').cursor()
+local system = uci:get_first('system', 'system')
+
+local f = Form(translate('Logging'), translate(
+	'If you want to use a remote syslog server, you can set it up here. '
+	.. 'Please keep in mind, that the data is transmitted unencrypted.'
+))
+local s = f:section(Section)
+
+local enable = s:option(Flag, 'log_remote', translate('Enable'))
+enable.default = uci:get_bool('system', system, 'log_remote')
+enable.optional = true
+function enable:write(data)
+	uci:set('system', system, 'log_remote', data)
+end
+
+local ip = s:option(Value, 'log_ip', translate('IP'))
+ip.default = uci:get('system', system, 'log_ip')
+ip:depends(enable, true)
+ip.optional = false
+ip.placeholder = '0.0.0.0'
+ip.datatype = 'ipaddr'
+function ip:write(data)
+	if data ~= nil then
+		uci:set('system', system, 'log_ip', data)
+	else
+		uci:delete('system', system, 'log_ip')
+	end
+end
+
+local port = s:option(Value, 'log_port', translate('Port'))
+port.default = uci:get('system', system, 'log_port')
+port:depends(enable, true)
+port.optional = true
+port.placeholder = 514
+port.datatype = 'uinteger'
+function port:write(data)
+	if data ~= nil then
```

Comment:
Ah, you're right, the set() interface is still confusing in simple-uci... I'll need to fix that in simple-uci, but we can keep the if/else here for now.